### DISCUSSION TITLE
Tree 组件的 selectedKeys和selectedKey、SelectedData、SelectedNode都是双向绑定吗

### DISCUSSION THREAD

**Discussion by moumousoup** [ant-design/ant-design#47933] [Q&A]
为什么我绑定无效？

**Comment by moumousoup** [comment]
我绑定任何一个都不起作用！！！！

**Comment by zombieJ** [comment]
React 里没有双向绑定的说法，取而代之的是受控，是不是提问错地方了？

**Reply by moumousoup** [reply]
没有 现象就是在组件绑定@bind-SelectedKeys="selectedKeys"，后台代码修改selectedKeys无效

**Reply by zombieJ** [reply]
你问的是 Ant Design Blazor，这个是社区做的生态。我们这边是 Ant Design React 库，提问错地方了哈。去这里提问：

https://github.com/ant-design-blazor/ant-design-blazor
